themes/horizon/javascripts/discourse/initializers/topic-list-columns.gjs

```diff
@@ -117,6 +117,7 @@ export default {
               metaKey: event.metaKey,
               shiftKey: event.shiftKey,
               button: event.button,
+              which: event.which,
```

Comment:
May as well pass this through to the new event:

https://developer.mozilla.org/en-US/docs/Web/API/UIEvent/which#value_for_mouseevent_non-standard